dockerfiles/Dockerfile.publisher

```diff
@@ -0,0 +1,27 @@
+# CI test image for unit/lint/type tests
+FROM node:lts-alpine
+
+RUN apk add --update bash git
+
+RUN rm -f ~/.npmrc
+
+ARG REG_BASIC_CREDENTIAL
+ARG REG_NPM_EMAIL
+RUN echo "_auth=$REG_BASIC_CREDENTIAL" >> ~/.npmrc
+RUN echo "email=$REG_NPM_EMAIL" >> ~/.npmrc
+RUN echo "always-auth=true" >> ~/.npmrc
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+
+COPY . .
+
+RUN npm run bootstrap
+RUN npm run build
+
+RUN git checkout .
```

Comment:
does this happen due to the `npm run bootstrap` command making changes to lockfiles?

If so it would be good to avoid that in the first place. You can add npm args to lerna with `--` so you could add a new command to `package.json`:

```
  "bootstrap:ci": "lerna bootstrap -- --no-save"
```

I think this should work, see [npm's docs](https://docs.npmjs.com/files/package-locks#using-locked-packages).